trajopt_utils/package.xml

```diff
@@ -13,6 +13,7 @@
   <build_depend>ros_industrial_cmake_boilerplate</build_depend>
   <build_depend>eigen</build_depend>
   <build_export_depend>eigen</build_export_depend>
+  <depend>libboost-program-options</depend>
```

Comment:
Does this need the -dev version?

Reply:
Good catch. I updated it to the following but going to wait for windows CI to finish because it is failing for some reason related to osqp with an odd error.

``` xml
  <build_depend>libboost-program-options-dev</build_depend>
  <exec_depend>libboost-program-options</exec_depend>
```